vllm/entrypoints/openai/serving_chat.py

```diff
@@ -94,19 +100,78 @@ def _parse_chat_message_content_parts(
         parts: Iterable[ChatCompletionContentPartParam],
     ) -> ChatMessageParseResult:
         texts: List[str] = []
+        image_futures: List[Awaitable[ImagePixelData]] = Field(
+            default_factory=list)
```

Comment:
See above

Reply:
I didn't read too carefully and looked at the wrong place...